<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>858</width>
    <height>508</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>858</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>人员管理</string>
    </property>
    <addaction name="actionStuInfo"/>
    <addaction name="actionTeaInfo"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>课程管理</string>
    </property>
    <addaction name="actionCourseInfo"/>
    <addaction name="actionSelectCourse"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStuInfo">
   <property name="text">
    <string>学生信息管理</string>
   </property>
  </action>
  <action name="actionTeaInfo">
   <property name="text">
    <string>教师信息管理</string>
   </property>
  </action>
  <action name="actionCourseInfo">
   <property name="text">
    <string>课程信息管理</string>
   </property>
  </action>
  <action name="actionSelectCourse">
   <property name="text">
    <string>选课管理</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
